.github/workflows/test.yml

```
@@ -0,0 +1,92 @@
+    - name: Check formatting
+      shell: pwsh
+      run: |
+        Write-Host "Checking code formatting..."
+        try {
+          cargo fmt -- --check
+          Write-Host "✅ Code formatting is correct"
+        }
+        catch {
+          Write-Host "❌ Code formatting issues found"
+          throw $_.Exception.Message
+        }
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)